{"name": "test_forgotpassword_resend_function", "status": "failed", "statusDetails": {"message": "AssertionError: assert None == 'SUCCESS'", "trace": "driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"2f67207da542d0f744af9b96155502f8\")>\n\n    @pytest.mark.usefixtures(\"driver\")\n    def test_forgotpassword_resend_function(driver):\n        driver.get(config[\"base_url\"])\n        captch_url= config[\"captch_url\"]\n        username = config[\"forgotpassword\"][\"email_5times\"]\n        forgotpass = Forgotpassword(driver)\n        forgotpass.click_forgot_password_link()\n        forgotpass.create_username(username)\n        forgotpass.click_continue_button()\n        print(\"waiting for 30 sec\")\n        time.sleep(31)\n        forgotpass.click_resend_otp_link()\n        time.sleep(3)\n        forgotpass.capture_otp(username,captch_url)\n        time.sleep(2)\n        driver.save_screenshot(\"toast_debug2.png\")\n        # driver.save_screenshot(\"toast_debug.png\")\n        title, desc = forgotpass.get_toast_message()\n>       assert title == \"SUCCESS\"\nE       AssertionError: assert None == 'SUCCESS'\n\ntests\\test_forgotpassword.py:135: AssertionError"}, "attachments": [{"name": "stdout", "source": "703d4dbe-3ac9-45f1-859c-74f4e6db00dd-attachment.txt", "type": "text/plain"}], "start": 1757590636640, "stop": 1757590719961, "uuid": "68f1b887-d962-4433-93f8-b2fbb2d47b51", "historyId": "fe726bcfc909283923f1299e44bd99dd", "testCaseId": "fe726bcfc909283923f1299e44bd99dd", "fullName": "tests.test_forgotpassword#test_forgotpassword_resend_function", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_forgotpassword"}, {"name": "host", "value": "OTSI-Lap80"}, {"name": "thread", "value": "84704-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_forgotpassword"}]}